### ISSUE TITLE
Bluetooth Barcode Scanner issues

### ISSUE THREAD

**Issue by robmacecs** [ionic-team/cordova-plugin-ionic-webview#36]
Okay, this one is a bit strange. After installing this plugin, using a barcode scanner in any text input ends up with the keys firing in the wrong order.

AFAIK a bluetooth barcode scanner works just like keyboard, that quickly types the characters followed by a return. (I think this weird delay-behaviour also happens if you type quickly, something about the return key responding faster than the other keys?)

Basically, if you do this in a text area, you will notice that the characters are all there but that, there is a new line in the middle of the word rather than the end. This is reproducibly due to installing this plugin (removing fixes it, reinstalling breaks it again)

Does anyone have any ideas? 

IOS 11, latest version of this plugin
